boards/mc-pic32mx-lccbuffer/ISRwrapper.inc

```diff
@@ -1 +0,0 @@
-../microchip-pic32mx-generic/ISRwrapper.inc
```

Comment:
See previous comment. I think I know what happened here. My local master still has the old links for the duinomitemega, where ISRwrapper.inc is located in the application target directory. Probably that board didn't get updated while checking out the pic32mx-ram-optimize branch.